OpenOversight/app/templates/index.html

```diff
@@ -23,9 +23,7 @@ <h1>OpenOversight</h1>
         </div>
         <div class="row display-flex horizontal-padding text-start">
           <div class="col-lg-6">
-            <svg xmlns="http://www.w3.org/2000/svg"
-                 version="1.1"
```

Comment:
The `version` property has been deprecated: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/version